src/modules/dashboard/components/EditColonyDetailsDialog/EditColonyDetailsDialog.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { FormikProps } from 'formik';
+import * as yup from 'yup';
+
+import Dialog, { DialogProps } from '~core/Dialog';
+import { ActionForm } from '~core/Fields';
+
+import { Colony } from '~data/index';
+import { ActionTypes } from '~redux/index';
+import { WizardDialogType } from '~utils/hooks';
+
+import DialogForm from './EditColonyDetailsDialogForm';
+
+export interface FormValues {
+  name: string;
+  annotation: string;
+}
+
+interface CustomWizardDialogProps {
+  prevStep: string;
+  colony: Colony;
+}
+
+type Props = DialogProps & WizardDialogType<object> & CustomWizardDialogProps;
+
+const displayName = 'dashboard.EditColonyDetailsDialog';
+
+const EditColonyDetailsDialog = ({
+  cancel,
+  close,
+  callStep,
+  prevStep,
+  colony,
+}: Props) => {
+  const validationSchema = yup.object().shape({
+    name: yup.string().required(),
```

Comment:
I think the name input will have some sort of limit (at least on the client side)

Would you mind finding this out and putting it in (if any)

For reference the domain name was capped at 20 chars.